beetmoverscript/test/test_work_dir/task.json

```diff
@@ -19,6 +19,7 @@
             "public/build/target.txt",
             "public/build/target.mozinfo.json",
             "public/build/target_info.txt",
+            "public/build/target.test_packages.json",
```

Comment:
It looks like we have to populate `test_work_dir/cot/eSzfNqMZT_mSiQQXu8hyqg/public/build/target.test_packages.json` for `test_move_beets` to work.